File: Protocal_go/Net_Login.go
package pb
/**
由 Net_Login.proto 文件生成 ...
author:yh  2024.11.27
*/
 type Base_Login_LoginInfo struct {
  DeviceId   string
 }

 type Net_Login_LoginReq struct {
  DeviceId   string
  LoginListInfo   []*Base_Login_LoginInfo
  LoginMapInfo map[int32]*Base_Login_LoginInfo 
  LoginInfo   *Base_Login_LoginInfo
 }

 type Net_Login_LoginRet struct {
  Ret   int32
 }

File: Protocal_java/Net_Login.java
package pb;
/**
由 Net_Login.proto 文件生成 ...
author:yh  2024.11.27
*/
 class Base_Login_LoginInfo {
   public String DeviceId;
 }

 class Net_Login_LoginReq {
   public String DeviceId;
   public Base_Login_LoginInfo []LoginListInfo;
   public map[int]Base_Login_LoginInfo LoginMapInfo; 
   public Base_Login_LoginInfo LoginInfo;
 }

 class Net_Login_LoginRet {
   public int Ret;
 }

